resources/qw-fields.txt

```diff
@@ -0,0 +1 @@
+software-component.id,software-component.name,software-component.description,software-component.safety-classification,soup.id,soup.name,soup.description,hazardous-situation.id,hazardous-situation.name,hazardous-situation.description,specific-software-cause.id,specific-software-cause.name,specific-software-cause.description,specific-software-cause.hazardous-situation,specific-software-cause.component,verification.id,verification.name,verification.description,verification.of,validation.id,validation.name,validation.description,validation.of,verification-result.version,verification-result.date,verification-result.runner,verification-result.raised-issues,verification-result.verification,validation-result.version,validation-result.date,validation-result.runner,validation-result.raised-issues,validation-result.validation,change-request.id,change-request.name,change-request.description,change-request.problem-report,change-request.approved,change-request.approver,change-request.approval-date,change-request.approval-reason,problem-report.id,problem-report.name,problem-report.description,problem-report.risk-assessment,problem-report.approved,problem-report.approver,problem-report.approval-reason,problem-report.approval-date,software-requirement.id,software-requirement.name,software-requirement.description,software-requirement.category,software-requirement.system-requirement,system-requirement.id,system-requirement.name,system-requirement.description,risk-control-measure.id,risk-control-measure.name,risk-control-measure.description,risk-control-measure.specific-software-cause,common-software-defect.id,common-software-defect.name,common-software-defect.description,common-software-defect.risk-control-measure
```

Comment:
is it worth keeping this to only be what we're using at the moment?

Reply:
Yes, it's not very useful. We need it to tell Word what merge fields to allow. I had an idea to auto-generate it, but the current implementation doesn't really allow that. I'm wondering whether to force the user to be explicit about what links to what (there are several advantages to this), in which case we might be able to make a command that generates it, which would be more useful to the user.

Reply:
I have deleted it in the follow-up because I have a command to produce it from the actual data structures involved.